bot.py

```
@@ -12,10 +12,15 @@
 # Helper function to send "..." before any message
 async def send_typing_indicator(chat_id, bot):
-    ellipsis = await bot.send_message(chat_id, "...")
+    dots = await bot.send_message(chat_id, "...")
     await asyncio.sleep(1)
-    await bot.delete_message(chat_id, ellipsis.message_id)
+    await bot.delete_message(chat_id, dots.message_id)
 
+
+def remember_name(name, user):
+    with open('storage.json', 'a+') as file:
+        for line in file:
```

Ruff (0.12.2)

Loop control variable `line` not used within loop body

Rename unused `line` to `_line`

(B007)